Code/Framework/AzCore/AzCore/Serialization/Json/JsonSerializationResult.cpp

```diff
@@ -10,265 +10,263 @@
 #include <AzCore/Serialization/Json/JsonSerialization.h>
 #include <AzCore/std/algorithm.h>
 
-namespace AZ
+namespace AZ::JsonSerializationResult
 {
-    namespace JsonSerializationResult
+    namespace Internal
```

Comment:
How about opening up two namespaces. 
The `namespace AZ::JsonSerializationResult::Internal` here and then on line 130 close the namespace and open up the namespace of `namespace AZ::JsonSerializationResult`

That would allow all the functions to have a single indentation for the entire file